build.sbt

```diff
@@ -117,12 +118,17 @@ lazy val pureconfigDependencies = Seq(
   "com.github.pureconfig" %% "pureconfig" % versions.pureconfig
 )
 
+lazy val monocleDependencies = Seq(
+  "com.github.julien-truffaut" %% "monocle-core" % versions.monocle,
+  "com.github.julien-truffaut" %% "monocle-macro" % versions.monocle
+).map(_ % "test")
+
 // -----------------
 
 lazy val sharedDependencies = Seq(
   "com.github.scopt" %% "scopt" % "4.0.0-RC2",
   "joda-time" % "joda-time" % "1.6"
-) ++ circeDependencies ++ catsDependencies ++ loggingDependencies
+) ++ circeDependencies ++ catsDependencies ++ loggingDependencies ++ monocleDependencies
```

Comment:
I'm not sure we should put it everywhere by default. I would rather go with node for now.

Reply:
Yep, I wasn't sure as well if that was a good decision. I will change it to node only.